src/commands/deployWorkspaceProject/DeployWorkspaceProjectConfirmStep.ts

```diff
@@ -51,6 +51,7 @@ export class DeployWorkspaceProjectConfirmStep extends AzureWizardPromptStep<Dep
         );
 
         ext.outputChannel.appendLog(outputMessage);
+        context.telemetry.properties.confirmedResourceCreation = 'true';
```

Comment:
I don't think we explicitly need this because we can just check cancelled events and see if the step was `DeployWorkspaceProjectConfirmStep`